### ISSUE TITLE
[Feature] Include speed in GPX file

### ISSUE THREAD

**Issue by redmond2742** [abinpaul1/GPS-Video-Logger#7]
Since the app records [GPX 1.0](https://www.topografix.com/GPX/1/0/gpx.xsd), can it also log the speed?

**Comment by abinpaul1** [comment]
Yes, that is definitley something that can be calculated and added to the GPX file. I'll keep it as enhancement for now. 

Reference
https://github.com/flyingrub/SpeedMeter
